content/docs/16-component.md

```diff
@@ -74,6 +84,8 @@ This page lists the version details of various Palette components with respect t
 
```

Comment:
Should PCG images be grouped under a heading 1 as follows? That categorizes and allows spelling out PCG.

# Private Cloud Gateway Images
## vSphere PCG Image Version
## MAAS PCG Image Version
## OpenStack PCG Image Version